superdesk/users/services.py

```diff
@@ -346,6 +346,11 @@ def get_role(self, user):
     def set_privileges(self, user, role):
         user['active_privileges'] = get_privileges(user, role)
 
+    def get(self, req, lookup):
+        if req is not None and 'is_author' in req.args and req.args['is_author'] in ('0','1'):
+            lookup['is_author'] = bool(int(req.args['is_author']))
```

Comment:
this assumes `is_author` is present on all users, but it won't return existing users without that flag.

Reply:
Hi, there is a default value of "True" for this field, so it should work.

Reply:
yep but would it work for users created before?

Reply:
indeed it would not work in this case. But I think in this case it should be handled by an update script. Do you think it should be handled differently?

Reply:
yep that's another option, you can either do the lookup like `{'$eq': {'is_author': True}}` / {`'$neq': {'is_author': True}}` or add the update script

Reply:
after talking with @hlmnrmr we decided that I'll go with the upgrade script